api/src/main/java/jakarta/servlet/http/Cookie.java

```diff
@@ -503,6 +503,8 @@ private void putAttribute(String name, String value) {
      *
      * @param name the name of the cookie attribute to set the value for, case insensitive
```

Comment:
```suggestion
     * @param name the name of the cookie attribute to get the value of, case insensitive
```

Reply:
Tx. I've updated the PR to include this correction.